src/inmanta/data/__init__.py

```diff
@@ -2335,6 +2335,7 @@ def validate_cron(value: str, allow_empty: bool = True) -> str:
 AUTOSTART_AGENT_DEPLOY_SPLAY_TIME = "autostart_agent_deploy_splay_time"
 AUTOSTART_AGENT_REPAIR_INTERVAL = "autostart_agent_repair_interval"
 AUTOSTART_AGENT_REPAIR_SPLAY_TIME = "autostart_agent_repair_splay_time"
```

Comment:
@wouterdb do you know if this option is still relevant?

Reply:
it isn't